adserver/views.py

```diff
@@ -159,8 +162,11 @@ class AdvertiserMainView(
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        # Get the beginning of the month so we can show month-to-date stats
+        # Get the beginning/end of the month so we can show month-to-date stats
         start_date = timezone.now().replace(day=1, hour=0, minute=0, second=0)
+        end_date = start_date.replace(
+            day=calendar.monthrange(start_date.year, start_date.month)[1]
```

Comment:
Huh, didn't know about this. Time is hard. I wish there was a more explicit way to get the data here other than [1], but alas.

Reply:
Ya, I don't love it either.